modules/server/templates/configs/forseti_conf_server.yaml.tpl

```diff
@@ -301,6 +303,12 @@ notifier:
                 data_format: csv
                 # gcs_path should begin with "gs://"
                 gcs_path: gs://${FORSETI_BUCKET}/scanner_violations
+            %{ if VIOLATIONS_SLACK_WEBHOOK != "" }
```

Comment:
Should we add a `AUDIT_LOGGING_VIOLATIONS_SLACK_WEBHOOK` variable like on line 288 ? Same for all other webhooks

Reply:
i was hoping to void webhooks for each resource type. seems like an excessive # of variables and i don't see the use case for having resource specific webhooks.